setup.py

```diff
@@ -24,6 +24,9 @@
 data_files.append(("/usr/bin",  ["src/bin/image-minimizer",
                                  "src/bin/mk-s390-cdboot",
                                  "src/bin/composer-cli"]))
+data_files.append(("/usr/lib/lorax", ["src/modules/io.weldr.rpm",
```

Comment:
I'm not a fan of reverse domain names, when the project moves you end up either changing them and breaking things, or leaving them and confusing people. I'd rather use something like the way git does it, define a core set of commands, and a contribution set, and people just need to make sure not to stomp on each other's names.
eg. /usr/share/lorax/core/ and /usr/share/lorax/contrib or something similar.

Also, I think we should avoid calling them modules to prevent confusion with modularity. These will be stand-alone programs, or extensions, to lorax.

Reply:
They're reverse domain name to allow drop-in from different packages. I've been bitten too many times by conflicting names for these kinds of things :)

"modules" is an unfortunate name, I agree. How about "stages"?